core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/ApiServiceSpec.scala

```diff
@@ -355,25 +360,18 @@ trait ApiServiceSpec
 
     override val workspaceServiceConstructor = WorkspaceService.constructor(
       slickDataSource,
-      methodRepoDAO,
-      new HttpExecutionServiceDAO(mockServer.mockServerBaseUrl, workbenchMetricBaseName = workbenchMetricBaseName),
       executionServiceCluster,
-      execServiceBatchSize,
       workspaceManagerDAO,
       leonardoService,
-      methodConfigResolver,
       gcsDAO,
       samDAO,
       notificationDAO,
       userServiceConstructor,
-      genomicsServiceConstructor,
       maxActiveWorkflowsTotal,
```

Comment:
I don't think that `maxActiveWorkflowsTotal` and `maxActiveWorkflowsPerUser` are needed anymore, as the only usage (as far as I can tell) was moved to the SubmissionsService.

Reply:
TY